src/webview_provider/datapilotPanel.ts

```diff
@@ -166,6 +207,31 @@ export class DataPilotPanel extends AltimateWebviewProvider {
         );
         break;
 
+      case "file:replace-contents":
+        if (!window.activeTextEditor || !params.sql || !params.filePath) {
+          return;
+        }
+        this.dbtTerminal.debug(
+          "file:replace-contents",
```

Comment:
Very cryptic name of debug message

Reply:
renamed :)

Reply:
It's still very generic. I really suggest to always use class name+methodname so we know where this error comes from.